File: FCUBEAPP/src/app/models/tripsummaryrptlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Tripsummaryrptmodel } from "./tripsummaryrptmodel";

export class Tripsummaryrptlistmodel {
    tripSummaryRptlist: Tripsummaryrptmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}

File: FCUBEAPI/FleetTrans/Models/TripSummaryRptListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FleetTrans.Models
{
    public class TripSummaryRptListModel
    {
        public List<TripSummaryRptModel> TripSummaryRptlist { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}
